Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+
+## Rbenv & Ruby part
+
```

Comment:
I could't use the `ruby-2.1.5` Docker image as base because it is based on an old Ubuntu which is not maintained anymore.

Reply:
I did a quick try the other day coming from the gathering and I think I did manage to get that image running. However, what you mention of being out-of-date is a valid point. However, I'd like to try starting from an Alpine Linux because it'll lead to a much more lightweight image.

Reply:
Let's try with Alpine then, I was trying to use the images the closest to our prod, but lightness is a big concern too.

Reply:
> I have tried to use alpine but it is giving me issues with PhantomJS and also rbenv compilation.

I don't think it's worth spending more time getting Alpine to work. It's only for elite Docker-Hipsters anyway. :wink: